src/lib/components/button/button.module.scss

```diff
@@ -1,13 +1,15 @@
 @import '../../styles/variables/variables.module.scss';
 @import '../../styles/themes/themes.module.scss';
 
-.container {
-  margin: 0 auto;
-  text-align: center;
-  float: left;
-  width: 100%;
-  position: relative;
-}
+// this should be moved into a pattern... 
+
+// .container {
```

Comment:
could we delete this commented code? Or we need for latter?

Reply:
This is commented out because that's what we're questioning whether we can leave out or have to put back in